tests/arp/test_stress_arp.py

```diff
@@ -30,6 +31,34 @@
 }
 
 
+@pytest.fixture(autouse=True)
```

Comment:
I think the function level autoused fixture is more meaningful

Reply:
Could you clarify what change you prefer please?

Reply:
There are two test cases in this script, I think we should do such "deep clean" after each test case.

Reply:
That is already done within the test case - not using a fixture because the test logic is in a loop, it could be ran more than once in a single test function call.

https://github.com/sonic-net/sonic-mgmt/blob/5db63ac44beb205d805d2a8c05be20b8d6d19fac/tests/arp/test_stress_arp.py#L209
https://github.com/sonic-net/sonic-mgmt/blob/5db63ac44beb205d805d2a8c05be20b8d6d19fac/tests/arp/test_stress_arp.py#L125

These 2 can be encapsulated into a function and the tests (and clear fixture) could call that function instead to have less repeated code.